Form-Controls/index.html

```diff
@@ -9,19 +9,74 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+  <!-- <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script src="index.js"></script> -->
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <form name="choosing" method="post" class="field-wrap">
+      <div class="formbox">
+      <div>
+          <label for="name">Name</label>
+          <input type="text" id="name" name="name" required> 
+        </div>
+        <div>
+          <label for="family" >Family</label>
+          <input type="text" id="family" name="family" required> 
+        </div>
+        <div>
+          <label for="email">Email</label>
+          <input type="email" id="email" name="email" required> 
+        </div>
+          <div class="address">
+              <div>
+                  <label for="address">Address</label>
+                  <input type="text" id="address" name="address" required> 
+              </div>
+              <div class="postcode">
+                  <label for="postcode">Postcode</label>
+                  <input type="text" id="postcode" name="postcode" required> 
+              </div>
```

Comment:
Do we need this information? If so, how should we validate it (e.g. Postcodes have a certain format)

Reply:
I think for send Item we need address but for validate I can add patern on inpute for the check 
Post code:pattern="^[A-Z]{1,2}\d[A-Z\d]? \d[A-Z]{2}$" like this